yarpl/test/Observable_test.cpp

```diff
@@ -386,6 +386,16 @@ TEST(Observable, DISABLED_SimpleTake) {
   EXPECT_EQ(0u, Refcounted::objects());
 }
 
+TEST(Observable, IgnoreElements) {
+  ASSERT_EQ(0u, Refcounted::objects());
+
+  EXPECT_EQ(
+      run(Observables::range(0, 100)->ignoreElements()),
```

Comment:
nit: attach one more operator at the end of ignoreElements operator, like map operator.